packages/config/src/arbitrum-testnet.json

```diff
@@ -30,5 +30,33 @@
   "summoning_address": "0x9d012712d24C90DDEd4574430B9e6065183896BE",
   "summoning_start_block": 8568698,
   "treasure_address": "0x6333F38F98f5c46dA6F873aCbF25DCf8748DDc2c",
-  "treasure_start_block": 8568541
+  "treasure_start_block": 8568541,
+
+  "seed_of_life_address": "0x3956C81A51FeAed98d7A678d53F44b9166c8ed66",
```

Comment:
Instead of putting mainnet addresses in the testnet config, maybe we can just change the `prepare` commands for these subgraphs with `arbitrum.json`? Because if there really is a dev subgraph, it will get deployed pointing to testnet with incorrect addresses.

Reply:
You want to push that change for me?

Reply:
Pushed

Reply:
> Because if there really is a dev subgraph, it will get deployed pointing to testnet with incorrect addresses.

Funny story, we are about to have a dev subgraph for the marketplace. But I want to avoid deploying all the smolbrains stuff for now, lol.